### ISSUE TITLE
Donate fededp/syscalls-bumper to falcosecurity

### ISSUE THREAD

**Issue by FedeDP** [falcosecurity/evolution#209]
**Repository**:  

https://github.com/FedeDP/syscalls-bumper

**Motivation**

I'd like the SyscallsBumper to be donated to the falcosecurity.
We will use it on our test-infra cluster to periodically bump the list of supported syscalls by falcosecurity/libs.

Its aim is to update, through a periodic PR, these items:
* [syscall_table](https://github.com/falcosecurity/libs/blob/master/driver/syscall_table.c) adding newly discovered syscalls as `generic` events
* compat tables (https://github.com/falcosecurity/libs/blob/master/driver/syscall_compat_{aarch64,x86_64,s390x}.h)
* ppm_events_public.h PPM_SC enum values
* build a report on supported syscalls (through specific event, not the generic one) that will be used to build a github page stating the supported syscalls status.

Basically, with the tool, we are able to always support up-to-date syscalls at least as generic events, meaning that an user can use them to filter for event types.

EDIT: please note that the tool supports all of the above, but expects https://github.com/falcosecurity/libs/pull/649 to be merged.

**Comment by leogr** [comment]
Hey @FedeDP 

I love it :heart_eyes: 

Ofc, +1 from me

PS
The only thing I'd change is just the repo name style (`SyscallsBumper` :point_right: `syscalls-bumper`)

**Comment by FedeDP** [comment]
I think we can change it while donating, right?

**Comment by FedeDP** [comment]
Renamed. Works :)

**Comment by FedeDP** [comment]
See https://github.com/falcosecurity/test-infra/pull/883 for a new periodic prow job that is awaiting this donation :)

**Comment by maxgio92** [comment]
Love this automation @FedeDP! Thank you for the effort and willing to donate it

**Comment by jasondellaluce** [comment]
+1 from my end as well!

**Comment by leogr** [comment]
Hey @FedeDP 
I'll ping you soon, to help you with the contribution :partying_face:

**Comment by FedeDP** [comment]
Yay!! https://github.com/falcosecurity/syscalls-bumper

/close
